menuItems.h

```diff
@@ -84,14 +81,14 @@ void showHelp(){
 
 // Function to show the
 // about text for the game
-void showHelp(){
+void showAbout(){
     //this function shall be expanded later on;
     cout<<"This is the about text";
 }
 
 
-
-void showMenu(int selection[])
```

Comment:
Again, the way I intended to make it work, the selection array should be necessary.